documents/TestSuite/Utilities/Lights/light_rig.mtlx

```diff
@@ -1,29 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <materialx version="1.36">
-  <directionallight name="dirLight1" type="lightshader">
+  <directionallight name="headlight" type="lightshader">
```

Comment:
New light rig. Labelled the new lights to show the actual orientation we use (+Y up, -Z in, and +X right.